Assets/Scripts/Layers/Properties/TransformPropertySection.cs

```diff
@@ -25,42 +25,71 @@ private class SetOfXYZ
 
         private const string percentageCharacter = "%";
 
-        public override HierarchicalObjectLayerGameObject LayerGameObject
+        public override HierarchicalObjectLayerGameObject LayerGameObject //todo: possibly remove this and replace it with a direct reference to the TransformLayerProperty
```

Comment:
Is deze TODO nog van toepassing?

Reply:
ja, het zou mooi zijn om deze section niet afhankelijk te maken van het visualisatie object, maar vaan het property object, maar het is denk ik beter om dat niet in deze pr te doen